databricks_cli/workspace/api.py

```diff
@@ -166,15 +168,16 @@ def import_workspace_dir(self, source_path, target_path, overwrite, exclude_hidd
                                 'continue.').format(cur_src, extensions))
 
     def export_workspace_dir(self, source_path, target_path, overwrite, headers=None):
-        if os.path.isfile(target_path):
+        os_compatible_target_path = re.sub(LOCAL_OS_COMPATIBLE_PATH_REGEX, '_', target_path)
```

Comment:
This will replace `C:\\Users\\admin` with `C_Users\\admin`. Effectively it converts an absolute path to a relative path.